pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -712,12 +712,15 @@ func TestSyncMurStatusWithUserAccountStatuses(t *testing.T) {
 			HasFinalizer()
 
 		// Get the notification resource and verify it
-		notification := toolchainv1alpha1.Notification{}
-		err = hostClient.Get(context.TODO(), namespacedName(mur.Namespace, userAccount.Name+"-provisioned"), &notification)
+		notifications := &toolchainv1alpha1.NotificationList{}
+		err = hostClient.List(context.TODO(), notifications)
+		require.NoError(t, err)
+		require.Len(t, notifications.Items, 1)
+		notification := notifications.Items[0]
 		require.NoError(t, err)
 		assert.Equal(t, userAccount.Spec.UserID, notification.Spec.UserID)
 		assert.Equal(t, "userprovisioned", notification.Spec.Template)
-		assert.Equal(t, userAccount.Name+"-provisioned", notification.Name)
+		assert.Contains(t, notification.Name, userAccount.Name+"-provisioned")
```

Comment:
```suggestion
		assert.Contains(t, notification.Name, userAccount.Name+"-provisioned-")
		assert.True(t, len(notification.Name) > len(userAccount.Name+"-provisioned-"))
```

Reply:
Updated as part of: 21c323901080723156e76ab6563614f16880baf3